content/docs/reference/ios.md

```diff
@@ -41,6 +41,31 @@ WebAuthn credentials created using the platform authenticator in iOS/iPadOS 15 a
 <!-- TODO: cross link to generic content about "upgrading to a passkey" -->
 To replace a legacy platform credential with a passkey, start a credential registration ceremony and pass **the same user handle** (user.id) in the request. iOS/iPadOS will overwrite the legacy credential with a new passkey that will be backed up to iCloud Keychain.
 
+### WebViews
+
+#### Embedded WebViews
+
+`WKWebView` is the embedded WebView (EWV) on iOS and iPadOS. Embedded WebViews allow the calling app full control over the embedded web session, including modifying and intercepting requests, so many web platform features are limited in these contexts.
+
+> **NOTE:**<br>
+> Embedded WebViews run in the context of the calling app, meaning only passkeys for the linked web domain (RP ID) can be created or used for sign in.
+>
+> Said differently, only use EWV when sign in is handled by your own service (non-federated). When supporting multiple identity providers, System WebView should be used (see below).
+
+<a href="https://developer.apple.com/documentation/webkit/wkwebview" target="_blank"><button type="button" class="btn btn-light">WKWebView docs @ Apple Developer {{< icon-external-link size=24 >}}</button></a>
```

Comment:
Hey @g-davidson, I want to sanity check this with you: isn't `SFSafariViewController` now the preferred way to embed a WebAuthn-capable webview? I know it's newer, and that it handles more things that you'd have to manually implement with a `WKWebView`. 

I want to make sure we're suggesting the correct path forward for those orgs that can't implement ASWebAuthenticationSession.

Huh, there's an Apple Developer page about this:

https://developer.apple.com/news/?id=trjs0tcd

Anyway thanks for your feedback on this ✌️

Reply:
Good question! On iOS, ASWebAuthenticationSession is effectively a wrapper around SFSafariViewController, with the additional feature of being able to give you a callback URL (either using a custom scheme, or now using a Universal Link-like URL). ASWAS is generally easier to work with than SFSVC, and is the preferred path unless there's a specific reason it won't work.

The only reason I'm aware of that people might prefer SFSVC is because they couldn't use custom schemes for callbacks, but that should be solved now for new adoptions.

Reply:
Ah, but when choosing specifically between WKWebView or SFSafariViewController to **embed** a webview...is there an actual choice here? I'm trying to make this sound different than popping up ASWebAuthenticationSession - maybe it's an apples-to-oranges thing since SFSafariViewController opens as a separate view so it's not "embedding" in the traditional sense, vs embedding as in "wrapping a webview in custom chrome"...

This could just me by Apple development naivety on display 😅

Reply:
Yeah SFSafariViewController isn't really used for embedding; it's meant for doing things that need a full browser context. Depending on the platform, it either opens a modal view that covers the current app, or will also just go out to the full browser.